tests/mocks/ERC165.cairo

```diff
@@ -7,21 +7,17 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from openzeppelin.introspection.erc165.library import ERC165
 
 @view
-func supportsInterface{
-        syscall_ptr: felt*,
-        pedersen_ptr: HashBuiltin*,
-        range_check_ptr
-    } (interfaceId: felt) -> (success: felt):
-    let (success) = ERC165.supports_interface(interfaceId)
-    return (success)
-end
+func supportsInterface{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
Should we refactor to avoid returning tuples but single values in cases like this? (0.10 introduction of single return values)

Reply:
Ahh yes, good call. This is just what `cairo-migrate` spit out. I'll update